.github/CODEOWNERS

```diff
@@ -52,3 +52,4 @@ ESIMD/ @intel/dpcpp-esimd-reviewers
 esimd/ @intel/dpcpp-esimd-reviewers
 sycl/include/sycl/ext/intel/experimental/esimd.hpp @intel/dpcpp-esimd-reviewers
 sycl/doc/extensions/experimental/sycl_ext_intel_esimd/ @intel/dpcpp-esimd-reviewers
+llvm/lib/SYCLLowerIR/CMakeLists.txt @intel/dpcpp-esimd-reviewers
```

Comment:
@pvchupin, could you clarify the reasons to replace the reviewers of this file from @intel/dpcpp-tools-reviewers to @intel/dpcpp-esimd-reviewers?
According to my understanding SYCLLowerIR was originally used introduced for ESIMD passes, but eventually some non-ESIMD passes were moved to this library as well, so I don't think we should remove @intel/dpcpp-tools-reviewers from reviewers.
If we need to notify ESIMD team about future changes in this file, I suggest keep @intel/dpcpp-tools-reviewers and add @intel/dpcpp-esimd-reviewers.

Reply:
That's fine to me. Thanks.
@kbobrovs also mentioned that ESIMD parts in the future could be moved to ESIMD specific cmake...